.github/workflows/cfc-validation.yml

```
@@ -68,6 +68,20 @@ jobs:
           cdk --version
           echo "AWS CDK installed"
 
+      - name: Install Rust toolchain (for cfn-guard)
+        uses: actions-rs/toolchain@v1
```

actionlint (1.7.9)

the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)